FreeRTOS/Demo/CORTEX_M3_MPS2_QEMU_GCC/scripts/mps2_m3.ld

```diff
@@ -75,6 +72,10 @@ SECTIONS
 		. = ALIGN(4);
 	} >FLASH
 
+	. = ALIGN(4);
```

Comment:
How about keeping the `_etext` where it is and define `_sidata` right before `.data` section like the following:

```
_sidata = LOADADDR(.data);
```

Reply:
Got it~
I will modify it.